packages/cfpb-design-system/src/components/cfpb-tooltips/index.js

```diff
@@ -25,7 +25,7 @@ function Tooltip(element) {
    * @returns {Tooltip} An instance.
```

Comment:
This doc isn't quite right, is it? The returned type is whatever `tippy` returns, not an instance of `Tooltip`?